classes/Deck.js

```diff
@@ -0,0 +1,665 @@
+
+
+export default class Deck {
+  constructor() {
+    this.cards = [
+      {
+        image: require("../components/cards/png/1.png"),
+        number: 1,
+        color: "Red",
+        shape: "Squiggle",
+        shading: "Solid",
+        selected: false
+      },
```

Comment:
The list of cards takes up most of this class file, which is fine but I would think about separating that data out into its own JSON file - that way the class constructor is easier to read and when something changes in the card data you don't have to touch the constructor.